frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -675,6 +699,15 @@ function ReaderBookmark:onShowBookmark()
         item.type = self.getBookmarkType(item)
         if not self.match_table or self:doesBookmarkMatchTable(item) then
             item.text = self:getBookmarkItemText(item)
+            if bookmarks_items_show_color then
+                if item.color == nil or item.color == "gray" then
+                    if bookmarks_items_show_gray then
```

Comment:
Is "gray" our default everywhere ? On my emulator, I get my highlights in yellow since a few months ago. I don't remember having done anything special to get them yellow (?).